www/app/hardware/Hardware.html

```diff
@@ -1522,6 +1522,13 @@ <h2><span data-i18n="Hardware">Hardware</span>:</h2>
 					&#8592;&nbsp;<span data-i18n="Leave empty for"></span>&nbsp;<b>domoticz/out</b>
 				</td>
 			</tr>
+			<tr>
+				<td align="right"><label for="mqttdiscoveryprefix"><span data-i18n="Auto Discovery  Prefix"></span>:</label></td>
```

Comment:
Double space between Discovery and Prefix

Reply:
Thanks fixed. The reason for the .exe is that it was not intended, but I updated the transifex tools... not a big problem but it should not have been here